plugins/modules/aci_tenant_action_rule_profile.py

```diff
@@ -249,18 +361,64 @@ def main():
             module_object=action_rule,
             target_filter={"name": action_rule},
         ),
+        child_classes=list(child_classes.keys()),
     )
 
     aci.get_existing()
 
     if state == "present":
+        child_configs = []
+        for key, value in child_classes.items():
+            if value[0] is not None:
```

Comment:
the conditional is nested and quite complex, perhaps add some comments to make the conditional more understandable